src/SmartComponents/RunTaskModal/SystemsSelect.js

```diff
@@ -0,0 +1,121 @@
+import React, { useState } from 'react';
+import propTypes from 'prop-types';
+import { Alert } from '@patternfly/react-core/dist/js/components/Alert';
+import { Flex, FlexItem } from '@patternfly/react-core/dist/js/layouts/Flex';
+import {
+  Form,
+  FormGroup,
+} from '@patternfly/react-core/dist/js/components/Form';
+import { TextInput } from '@patternfly/react-core/dist/js/components/TextInput';
+import { ValidatedOptions } from '@patternfly/react-core/dist/js/helpers/constants';
+import SystemTable from '../SystemTable/SystemTable';
+import {
+  AVAILABLE_TASKS_ROOT,
+  INFO_ALERT_SYSTEMS,
+  TASKS_API_ROOT,
+} from '../../constants';
+import ReactMarkdown from 'react-markdown';
+import warningConstants from '../warningConstants';
+import { useSystemBulkSelect } from './hooks/useBulkSystemSelect';
+
+const SystemsSelect = ({
+  createTaskError,
+  description,
+  selectedIds,
+  setSelectedIds,
+  setTaskName,
+  slug,
+  taskName,
+}) => {
+  const warningConstantMapper = `${slug
+    ?.toUpperCase()
+    .replace(/-/g, '_')}_WARNING`;
+
+  const [filterSortString, setFilterSortString] = useState('');
+
+  const { bulkSelectIds, selectIds } = useSystemBulkSelect(
+    selectedIds,
+    setSelectedIds,
+    filterSortString
+  );
+
+  return (
+    <React.Fragment>
+      <Flex>
+        <FlexItem>
+          <b>Task description</b>
+        </FlexItem>
+      </Flex>
+      <Flex style={{ paddingBottom: '8px' }}>
+        <FlexItem style={{ width: '100%' }}>
+          <ReactMarkdown>{description}</ReactMarkdown>
+        </FlexItem>
+      </Flex>
+      <Flex>
+        <FlexItem>
+          <a href={`${TASKS_API_ROOT}${AVAILABLE_TASKS_ROOT}/${slug}/playbook`}>
```

Comment:
This is more for cleanliness, you dont have to have a <flexItem> elem with something inside, this could be simplified to just the <a> tag